haiku-apps/genio/genio-2.1.0.recipe

```diff
@@ -29,7 +29,7 @@ PROVIDES="
 REQUIRES="
 	haiku${secondaryArchSuffix}
 	gcc${secondaryArchSuffix}_syslibs_devel
-	llvm17${secondaryArchSuffix}_clang
+	cmd:clang$secondaryArchSuffix
```

Comment:
should be cmd:clang

Reply:
My bad, @jackburton79 could you do the honours?

Reply:
Would this work on 32 bit for the secondary arch ?

Reply:
Resolved, thanks! :)

Reply:
> Would this work on 32 bit for the secondary arch ?

Genio is already on secondary arch for 32bit, cmd:clang also, but installed to $prefix/bin instead of $binDir ($prefix/bin/x86).